intranet/templates/lostfound/founditem_delete.html

```diff
@@ -0,0 +1,51 @@
+{% extends "page_with_nav.html" %}
+{% load static %}
+{% load dates %}
+{% load pipeline %}
+
+{% block title %}
+    {{ block.super }} - Delete {{ type|title }}
+{% endblock %}
+
+{% block js %}
+    {{ block.super }}
+{% endblock %}
+
+{% block css %}
+    {{ block.super }}
+{% endblock %}
+
+{% block head %}
+    {% if dark_mode_enabled %}
+        {% stylesheet 'dark/base' %}
+        {% stylesheet 'dark/nav' %}
+    {% endif %}
+{% endblock %}
+
+{% block main %}
+<div class="lostitem primary-content">
+    <a href="{% url 'lostfound' %}" class="button">
+        <i class="fas fa-arrow-left"></i> Lost and Found
+    </a>
+    <h2>
+        Delete {{ founditem.title }}
+    </h2>
+    <br>
+    
+    <b>Are you sure you want to delete {{ founditem.title }}?</b>
+    <form method="post" action="">
+        {% csrf_token %}
+        <input type="radio" name = "full-delete" value = "y">
+        <span>Full Delete</span>
+        <input type="radio" name = "mark_found" value = "">
+        <span>Mark Claimed</span>
```

Comment:
```suggestion
        <span>Mark Retrieved</span>
```
I know I said yes to "Claimed" in person, but then I realized that we use "retrieved" everywhere, and we should be consistent.